### DISCUSSION TITLE
闪退的问题- 请你们帮忙提交应用内报告. 在这里留言! 谢谢! (All logs here)

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#348] [Announcements]
你们的声音我们都听到了. 请给我们一点时间. 非常抱歉影响了体验感. 也谢谢你们支持蓝灯!

如果各位方便的话. 帮忙提交应用内报告. 然后把 8 位数票号给我们. 这会帮很大的忙.

打个比方:
票号 (Ticket number/ Log number) - 12345678
系统- Window 10, 11/ IOS xxx/ Mac OS xxx/ Android xxx?
版本- 7.8.3 / x. x. xxx

蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。

您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢您的等待与支持。

**Comment by jean-reno** [comment]
票号：70929826
系统：win10 LTSC
版本：7.8.1 (20240229.143638)

**Reply by MaxMKMax** [reply]
@jean-reno 谢谢您. 这会帮助到解决问题.

**Comment by taoistx** [comment]
支付宝商家订单号：src_1OTi9cFW8MAXbdooTOhmBDK3
系统：win11
版本：在哪里看？

**Reply by MaxMKMax** [reply]
@taoistx 版本在setting下方或角落.

**Comment by Alinezhao** [comment]
票号：70931308
系统：Mac os 12.7.4 
版本：7.8.3 

（在另外一台同样配置的Mac上和thinkpad win10上可以用）

**Comment by jean-reno** [comment]
目测 国内的区域不一样 给推的版本 也不一样 我的是7.8.1  你们怎么说是7.8.3 ？我根本都找不到这个版本

**Reply by chaojixuan** [reply]
确实，我这里也是只有7.81

**Comment by 1070046110** [comment]
票号：70954565
系统：Windows11
版本：7.8.3

**Reply by 1070046110** [reply]
删除了global文件之后可以打开了，不知道退出之后还能不能再打开。删除之后变成了普通版需要重新认证设备，而且感觉效果没有之前稳定。

**Comment by MaxMKMax** [comment]
嗯嗯. 谢谢各位的帮忙. 明天我会和科技团队开会. 我把你们所遇到的所有问题和他们谈和找解决方案!

**Comment by antheeee** [comment]
票号：70949871, 70985069
系统：Windows10
版本：7.8.1

**Comment by szr119** [comment]
票号：70983695 70953092
系统：windows 10
版本：7.8.1
